### ISSUE TITLE
text-muted contrast

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#47551]
### Steps to reproduce the issue

1. add the class text-muted to any element in the atum template

### Version

5.4

### Expected result

contrast that passes wcag aa

### Actual result

contrast is only 3.2:1
it should be at least 4.5:1

### What device were you using? (eg, phone, laptop)

desktop

### What web browser were you using? (eg, Chrome, Safari)

chrome Windows

### Additional Comments

This was raised previously by @chmst but closed #41196

There are also two competing css in atum (both fail) not sure why 

<img width="718" height="486" alt="Image" src="https://github.com/user-attachments/assets/9e7c3af2-9093-4166-b93f-cb8dc97f6714" />

it is ok in dark mode
